implementations/php/provekit-ir-symbolic/src/Ir/Formula.php

```
@@ -41,7 +41,7 @@ public function jsonSerialize(): array {
 
 // ---------- Builder helpers (kit primitives) ----------
 
-function Var(string $name, ?Sort $sort = null): IrTerm {
+function V(string $name, ?Sort $sort = null): IrTerm {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$sort'. (undefined)

(UnusedFormalParameter)